.clj-kondo/config.edn

```diff
@@ -128,10 +128,12 @@
      clojure.core.async/to-chan!
      clojure.core.async/to-chan!!
      metabase.channel.core/send!
-     metabase.models.notification/create-notification!
      metabase.driver.sql-jdbc.execute/execute-prepared-statement!
+     metabase.models.notification/create-notification!
      metabase.pulse.send/send-pulse!
      metabase.query-processor.store/store-database!
+     ;; Safe in tests, where we typically use a private, temporary index per thread.
+     metabase.search.core/reindex!
```

Comment:
Not sure if its worth having this here anyway, it doesn't transitively within helpers.

Arguably that's a bug to fix in the linter.